gen3/bin/s3.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+#
+# Describe and create s3 buckets
+#
+
+source "${GEN3_HOME}/gen3/lib/utils.sh"
+gen3_load "gen3/gen3setup"
+
+help() {
```

Comment:
name all these function `gen3_s3_...`, so we could potentially `source` this file in the future, and names won't collide ...